src/Commands/ProjectCreateCommand.php

```diff
@@ -368,20 +376,6 @@ function ($state) use ($siteDir, $source) {
             // It is not necessary to push to GitHub so soon, but it's helpful
             // for debugging et. al. to have the initial repo contents available.
```

Comment:
Did the early push of the code cause GitLab to fire off a build right away and fail?

It's okay to remove this if it does not work well, but we should also get rid of the comment if we do.

Reply:
Yes, this was causing GitLab to fire a build early and fail since it was missing the proper environment variables. Comment removed.